readme.md

```diff
@@ -72,6 +76,19 @@ $ cd project-repo
 $ git init project-repo
 $ git add .
 $ git merge juan/feature
+
+MY ANSWER
+$ git remote add origin git@github.com/username/project-repo.git
```

Comment:
I would be careful about the order here, this would be fine if we had cloned down prior, because then git would already be setup, but here we need to run `git init` first so that git is setup for this project